src/tools/brush.js

```diff
@@ -1044,6 +1045,12 @@ export class Brush extends EventTarget {
       sliceMeta
     );
 
+    // this.#displayMask causes the position to get reset,
```

Comment:
couldn't the set position be made in the `if (viewController.equalImageMeta(searchMaskMeta))` part that calls the `displayMask`?